gitpod-io/website
# :rocket: New Contact Confirmation page Layout

## Description



:rocket: New Contact Confirmation page Layout

## Related Issue(s)



Fixes #3500

## How to test


- Try to submit [`/contact/support`](https://deploy-preview-3514--gitpod-kumquat.netlify.app/contact/support) or [`/contact/sales`](https://deploy-preview-3514--gitpod-kumquat.netlify.app/contact/sales) & see :eyes:

## Release Notes



```release-note
NONE
```
